lib/appsignal/hooks/active_support_notifications.rb

```diff
@@ -0,0 +1,43 @@
+module Appsignal
+  class Hooks
+    class ActiveSupportNotificationsHook < Appsignal::Hooks::Hook
+      register :active_support_notifications
+
+      BANG = '!'.freeze
```

Comment:
I think it's recommended. It's a mutable value assigned to a constant. If you say `BANG << 'foo'` it doesn't raise an error or output a warning.

There's also a rubocop rule for it: https://github.com/bbatsov/rubocop/blob/49598a035cd39a75ec91466a4e1a79c7e4f0f5f9/lib/rubocop/cop/style/mutable_constant.rb